hazelcast/src/test/java/com/hazelcast/util/CollectionUtilTest.java

```diff
@@ -129,5 +193,51 @@ public void objectToDataCollection_deserializeBack() {
         }
     }
 
+    @Test
+    public void testToIntArray() {
+        Collection<Integer> list = new ArrayList<Integer>();
+        list.add(42);
+        list.add(23);
+        list.add(Integer.MAX_VALUE);
+
+        int[] intArray = toIntArray(list);
+
+        assertNotNull(intArray);
+        assertEquals(3, intArray.length);
```

Comment:
Since the value in the list are a bit unpredictable, I would test it like this:

```
assertEquals(list.get(0), intArray[0])
assertEquals(list.get(1), intArray[1);
assertEquals(list.get(2), intArray[2])
```

This makes it easier to read since you don't need to remember these values.